optuna/integration/allennlp.py

```diff
@@ -21,6 +21,32 @@
     TrackerCallback = object
 
 
+def _dump_best_config(config_file: str, study: optuna.Study) -> Dict:
+    best_params = study.best_params
+    for key, value in best_params.items():
+        best_params[key] = str(value)
+    config = json.loads(_jsonnet.evaluate_file(config_file, ext_vars=best_params))
+    return allennlp.common.params.infer_and_cast(config)
+
+
+def dump_best_config(input_config_file: str, output_config_file: str, study: optuna.Study) -> None:
+    """Save resulting jsonnet replacing masks with best params in the experiment.
```

Comment:
A suggestion on how to phrase it.
```suggestion
    """Save jsonnet after updating with parameters from the best trial in the study.
```